test/functional/wallet_labels.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test label RPCs.
+
+RPCs tested are:
+    - getlabeladdress
+    - getaddressesbyaccount
+    - listaddressgroupings
+    - setlabel
+    - sendfrom (with account arguments)
```

Comment:
This (and the line below) can be updated to `(with label arguments)`

Reply:
> This (and the line below) can be updated to (with label arguments)

This is actually intentional. `sendfrom` and `move` are account-specific and deprecated and won't take label arguments. Idea is that we support receiving *at* addresses but not really sending *from* addresses.

Reply:
ok, makes sense. I was confused by the code lower down in the test case:

```
        # Check that sendfrom label reduces listaccounts balances.
        for i, label in enumerate(labels):
            to_label = labels[(i+1) % len(labels)]
            node.sendfrom(label.name, to_label.receive_address, amount_to_send)
```